decisions/remote-server-backup.md

```diff
@@ -0,0 +1,57 @@
+# Remote Server Backup
+
+- *Date*: 2022-12-7
+- *Deciders*: 
+- *Status*: 
+- *Outcome*: 
+- *Related Material*: [Backup Issue](https://github.com/openmsupply/open-msupply/issues/767)
+
+## Context
+
+It's important to back up remote server data so that it can be restored in case of corruption/loss. This KDD is to discuss remote synchronised server backup.
+
+In existing 4d mSupply, for remote servers, we have backup files and current journal file. There is no way to re-initialise remote desktop mSupply sites, thus we must rely on local/dropbox backups. There have been cases with quite catastrophic outcomes where backup was restored incorrectly, causing major data synchronisation issues.
```

Comment:
Maybe misreading this, but you can recreate the datafile from the central server using [site export](https://docs.msupply.org.nz/synchronisation:site_export)! It can just be reaaaallly slow if it's a larger datafile...

Reply:
Thanks @DhanyaHerath, I should have mentioned this